### PR TITLE
[FIX] sale: use partner language for terms and conditions

### PR DESCRIPTION
**Pull request by Demesmaeker**
opw-3253362

### CODE REVIEW

**Review by vava-odoo — APPROVED** [review]

File: addons/sale/models/sale_order.py [hunk 1]
```diff
@@ -116,7 +116,9 @@ def _default_note(self):
         use_invoice_terms = self.env['ir.config_parameter'].sudo().get_param('account.use_invoice_terms')
         if use_invoice_terms and self.env.company.terms_type == "html":
             baseurl = html_keep_url(self._default_note_url() + '/terms')
+            context = {'lang': self.partner_id.lang or self.env.user.lang}
             return _('Terms & Conditions: %s', baseurl)
+            del context
```
Comment (vava-odoo) [inline addons/sale/models/sale_order.py:121]: You need to delete before returning

Comment (Demesmaeker) [reply addons/sale/models/sale_order.py:121]: xD Indeed

**Review by Feyensv — APPROVED** [review]
@robodoo delegate+

**Comment by Demesmaeker** [comment]
@robodoo r+

**Pull request closed without merging**